### PR TITLE
Fix TS errors in AttributeOverview

### PR DESCRIPTION
**Pull request by boyum**
I fucked up and commited the Markdown files early which are part of #35, SORRY

### CODE REVIEW

**Review by dagerikhl — APPROVED** [review]
Again, thanks @boyum , great work! :D

File: src/lib/components/attributes/attribute-overview/AttributeOverview.svelte [hunk 1]
```diff
@@ -41,16 +41,32 @@
...
+		for (let count = 0; count < POINTS_TO_DISTRIBUTE; count++) {
+			const randomAttribute = attributeNames[(Math.random() * attributeNames.length) | 0];
+			allocation[randomAttribute]++;
+		}
+
+		pointsRemaining = 0;
+		value = allocation;
+		displayedPointsRemaining.set(pointsRemaining);
+		displayedAttributes.set(allocation);
+	};
```
Comment (palexcast) [inline src/lib/components/attributes/attribute-overview/AttributeOverview.svelte:57]: This contains some of the solution code, but these two values are not set further up here.

```suggestion
	};
```

**Review by palexcast — CHANGES_REQUESTED** [review]
Contains some code that won't compile

**Review by palexcast — APPROVED** [review]
💚

**Pull request merged**